money/notsay.md

```diff
@@ -13,7 +13,7 @@ casual outsider, it appears as if I just had a period of extra motivation and
 energy and one fine day delivered this feature.
 
 Why companies would not want to use their sponsoring an Open Source project
-for marketing purposes and good will always baffles me. I have heard it
+for marketing purposes and good will always baffle me. I have heard it
```

Comment:
In this sentence "will" is a noun, not an auxiliary verb. The connection is not "will ... baffle", but "for [...] good will". https://en.wiktionary.org/wiki/good_will

Reply:
Reading your comment, I can see that argument, though I would counter that it's ambiguous in this sentence as to which it is, and it could go either way, and that 'good will' is a more uncommon form of use. If the intent was 'good will' as 'goodwill', then you are correct, but I'm not the original author and don't know what their intent was.

Reply:
> 'good will' is a more uncommon form of use.

I was trying to guess the intention from meaning, not frequency of usage. Sponsoring an Open Source project might indeed generate goodwill. In this sentence "goodwill" makes more sense to me than "for marketing purposes and good".